src/Microsoft.Azure.WebJobs.Extensions.Redis/RedisUtilities.cs

```diff
@@ -53,13 +57,16 @@ public static string ResolveConnectionString(IConfiguration configuration, strin
                 throw new ArgumentNullException(nameof(connectionStringSetting));
             }
 
-            string connectionString = configuration.GetConnectionStringOrSetting(connectionStringSetting);
+            IConfigurationSection section = configuration.GetWebJobsConnectionSection(connectionStringSetting);
+            string connectionString = section.Value;
             if (string.IsNullOrWhiteSpace(connectionString))
             {
                 throw new ArgumentOutOfRangeException($"Could not find {nameof(connectionStringSetting)}='{connectionStringSetting}' in provided configuration.");
             }
 
-            return connectionString;
+            options = ConfigurationOptions.Parse(connectionString);
+            options.ClientName = string.Format(RedisClientNameFormat, clientName);
```

Comment:
`string.Format()` is a deprecated pattern - can we eliminate the `RedisClientNameFormat` constant and use string interpolation here?